modules/op-running-as-root-user-namespace.adoc

```diff
@@ -0,0 +1,64 @@
+:_content-type: PROCEDURE
+
+[id='op-running-as-root-user-namespace_{context}']
+= Running as root in a user namespace
+
+When you run a command in a user namespace, your container runs as root (user id 0) but has user privileges on the host. {product-title} uses CRI-O as its container runtime interface. CRI-O has a workload feature to run a pod in the user namespace.
+
+With {pipelines-shortname}, to run pods in the user namespace, you must pass the annotations that CRIO expects. To add these annotations for all users, you use the `oc edit clustertask buildah` command to edit the `buildah` cluster task. Or, to add the annotations to a specific namespace, you export the cluster task as a task to that namespace.
```

Comment:
Seems to me like this paragraph could be a procedure. You're giving the user multiple actions to take to when using pipelines to run pods in the user namespace. 

I think this content would be more appropriate as procedure steps since it is very user action driven.

Reply:
Something link:

Use Pipelines to run pods in the user namespace to <state value of running pods in user namespace>. 

_Prerequisites_

- Pass the annotations that CRIO expects.

_Procedure_

1. Use the `oc edit clustertask buildah` command to edit the `buildah` cluster task.
2. Add the annotations for all users. 
3. Export the cluster task as a task to the namepsace.